docs/loading_environment_variables.md

```diff
@@ -0,0 +1,160 @@
+
+imgproxy can load environment variables from various sources such as:
+
+* [Local file](#local-file)
+* [AWS Secrets Manager](#aws-secrets-manager)
+* [AWS Systems Manager Parameter Store](#aws-systems-manager-parameter-store)
+* [Google Cloud Secret Manager](#google-cloud-secret-manager)
+
+## Local file
+
+You can create an [environment file](#environment-file-syntax) and configure imgproxy to read environment variables from it.
+
+* `IMGPROXY_ENV_LOCAL_FILE_PATH`: the path of the environmebt file to load
+
+## AWS Secrets Manager
+
+You can store the content of an [environment file](#environment-file-syntax) in AWS Secrets Manager secret and configure imgproxy to read environment variables from it.
+
+* `IMGPROXY_ENV_AWS_SECRET_ID`: the ARN or name of the secret to load
+* `IMGPROXY_ENV_AWS_SECRET_VERSION_ID`: _(optional)_ the unique identifier of the version of the secret to load
+* `IMGPROXY_ENV_AWS_SECRET_VERSION_STAGE`: _(optional)_ the staging label of the version of the secret to load
+* `IMGPROXY_ENV_AWS_SECRET_REGION`: _(optional)_ the region of the secret to load
+
+**📝 Note:** If both `IMGPROXY_ENV_AWS_SECRET_VERSION_ID` and `IMGPROXY_ENV_AWS_SECRET_VERSION_STAGE` are set, `IMGPROXY_ENV_AWS_SECRET_VERSION_STAGE` will be ignored
+
+### Set up AWS Secrets Manager credentials
+
+There are three ways to specify your AWS credentials. The credentials policy should allow performing the `secretsmanager:GetSecretValue` and `secretsmanager:ListSecretVersionIds` actions with the specified secret:
+
+#### IAM Roles
+
+If you're running imgproxy on an Amazon Web Services platform, you can use IAM roles to to get the security credentials to retrieve the secret.
+
+* **Elastic Container Service (ECS):** Assign an [IAM role to a task](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html).
+* **Elastic Kubernetes Service (EKS):** Assign a [service account to a pod](https://docs.aws.amazon.com/eks/latest/userguide/pod-configuration.html).
+* **Elastic Beanstalk:** Assign an [IAM role to an instance](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/iam-instanceprofile.html).
+
+#### Environment variables
+
+You can specify an AWS Access Key ID and a Secret Access Key by setting the standard `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables.
+
+``` bash
+AWS_ACCESS_KEY_ID=my_access_key AWS_SECRET_ACCESS_KEY=my_secret_key imgproxy
+
+# same for Docker
+docker run -e AWS_ACCESS_KEY_ID=my_access_key -e AWS_SECRET_ACCESS_KEY=my_secret_key -it darthsim/imgproxy
+```
+
+#### Shared credentials file
+
+Alternatively, you can create the `.aws/credentials` file in your home directory with the following content:
+
+```ini
+[default]
+aws_access_key_id = %access_key_id
+aws_secret_access_key = %secret_access_key
+```
+
+## AWS Systems Manager Parameter Store
+
+You can store multiple AWS Systems Manager Parameter Store entries and configure imgproxy to load their values to separate environment variables.
```

Comment:
```suggestion
You can store multiple AWS Systems Manager Parameter Store entries and configure imgproxy to load their values to separate environment variables.
```
instead of "load" maybe it should "assign"?

Reply:
the reason I suggest is because we deal with multiple environments and it sounds kind of odd in that context